README.md

```diff
@@ -88,6 +88,15 @@ MyApp::Application.configure do
 end
 ```
 
+You can keep original rails log by setting `keep_original_rails_log` to ture, and assign another logger for lograge to use:
```

Comment:
Modify this to read 'You can also keep the original (and verbose) Rails logger by following this configuration:'